.github/workflows/main.yml

```diff
@@ -0,0 +1,157 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  IMAGE: ras-rm-print-file
+  REGISTRY_HOSTNAME: eu.gcr.io
+  HOST: ${{ secrets.GOOGLE_PROJECT_ID }}
+  RELEASE_HOST: ${{ secrets.RELEASE_PROJECT_ID }}
+  CHART_DIRECTORY: _infra/helm/ras-rm-print-file
+  SPINNAKER_TOPIC: ${{ secrets.SPINNAKER_TOPIC }}
+  ARTIFACT_BUCKET: ${{ secrets.ARTIFACT_BUCKET }}
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
+          token: ${{ secrets.BOT_TOKEN }}
+
+      - name: Run Tests
+        run: |
+          make test
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '270.0.0'
```

Comment:
https://github.com/GoogleCloudPlatform/github-actions/tree/master/setup-gcloud#usage  Super minor but this is at version 290 now, should we update our thing to use it?